org.gnome.Cowsay.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "org.gnome.Cowsay",
```

Comment:
We usually don't allow `org.gnome` in app id unless it's part of `GNOME` group on gitlab.gnome.org. I guess that `org.gnome.gitlab.Cowsay` would be alright though.

Reply:
@barthalion Is good, I don't want to cause any confusion with the GNOME trademark. Should I make a new pull request afterwards using the new slug?